<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessInterface</class>
 <widget class="QWidget" name="ProcessInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CardWidget" name="ProcessCard">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>560</width>
     <height>460</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="TableWidget" name="info_table"/>
     </item>
     <item>
      <widget class="IndeterminateProgressBar" name="progressBar"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>60</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="PrimaryPushButton" name="start_process">
         <property name="text">
          <string>开始数据处理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PushButton" name="loadPushButton">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;一般会自动加载，如果没加载出来，可以手动试试&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>手动加载数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PushButton" name="clean_log">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>清除日志</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="1">
        <widget class="CheckBox" name="repeat_news">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>重复新闻占比</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="CheckBox" name="time_num_change">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>新闻时间数量变化图</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="CheckBox" name="country_count">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>国家在新闻中占比</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="CheckBox" name="hot_wordcloud">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>热点词云</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="CheckBox" name="predict_market">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>芯片市场、制造趋势</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="CheckBox" name="good_or_bad">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>芯片封锁后果预测</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="PrimaryPushButton" name="draw">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>开始绘图</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="PlainTextEdit" name="logPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>251</width>
     <height>460</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>欢迎使用数据处理功能！</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IndeterminateProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
